manim/mobject/mobject.py

```diff
@@ -51,11 +44,33 @@
 
 # TODO: Explain array_attrs
 
-Updater = Union[Callable[["Mobject"], None], Callable[["Mobject", float], None]]
-T = TypeVar("T", bound="Mobject")
-
 if TYPE_CHECKING:
+    from PIL.Image import Image
+
     from ..animation.animation import Animation
+    from ..camera.camera import Camera
+    from ..utils.bezier import Interpolable
+
+# Generic type
+T = TypeVar("T")
+
+Updater = Union[Callable[["Mobject"], None], Callable[["Mobject", float], None]]
+# Any subclass of Mobject. Most Mobject methods `return self` so this is used
+# so that e.g. Circle.scale(2) returns a Circle, not an Mobject.
+MOS = TypeVar("MOS", bound="Mobject")
```

Comment:
Maybe a better name for this? I mean it's not needed. Maybe MOB instead of MOS

Reply:
"MOS" was meant to be "MObject Subclass"...  Maybe that was not obvious - happy to change it! Why MOB? Just MOBject?